x-pack/plugins/security_solution/docs/prebuilt_rules_customization_rfc.md

```diff
@@ -0,0 +1,2867 @@
+      <th>Example field in import payload</th>
+      <th>Overwrite existing rule?</th>
+      <th>Output</th>
+    </tr>
+  </thead>
+  <tbody>
+
+  
+  <tr>
+    <td>Not installed</td>
+    <td>No</td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My custom rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My custom rule",
+  rule_source: {
+    type: "internal"
+  }
+} 
+</pre>
+      </td>
+    </tr>
+  
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My custom rule",
+} 
+</pre>
+    </td>
+    <td>No</td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My edited rule",
+} 
+</pre>
+    </td>
+    <td>
+      <b>No</b>
+    </td>
+    <td>
+      Rule import rejected<br> because of rule_id match
+    </td>
+    <tr>
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My custom rule",
+} 
+</pre>
+    </td>
+    <td>No</td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My edited rule",
+} 
+</pre>
+    </td>
+    <td>
+      Yes
+    </td>
+    <td>
+<pre>
+{
+  name: "My edited rule",
+  rule_source: {
+    type: "internal"
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+  <tr>
+    <td>
+      Not installed
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td><b>No</b></td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+      Prebuilt rule import rejected because no matching <br>
+       version found for existing rule_id security-asset
+      </td>
+    <tr>
+
+
+
+
+  <tr>
+    <td>
+      Not installed
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+
+  <tr>
+    <td>
+      Not installed
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: true,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+} 
+</pre>
+    </td>
+    <td><b>No</b></td>
+    <td>
+    Rule import rejected<br> because of rule_id match
```

Comment:
Is rule import rejected because of rule overwrite flag is set to false?

Reply:
Exactly. Added that to the message in the last column